MinimedKit/PumpEvents/PumpEvent.swift

```diff
@@ -21,3 +21,22 @@ public protocol PumpEvent : DictionaryRepresentable {
     }
     
 }
+
+public extension PumpEvent {
+    public func isDelayedAppend(withPumpModel pumpModel: PumpModel) -> Bool {
+                
+        switch self {
+        case let bolus as BolusNormalPumpEvent:
+            //Square boluses for 523's are appended as the beginning of the event
+            if pumpModel == .Model523 {
+                return bolus.type != .Square
+            }
+            
+            // Square bolus' for some devices are delayed append
+            return bolus.type == .Square
```

Comment:
This should just be `return true`.  All pumps have delayed append on normal boluses.